examples/busybox/snapcraft.yaml

```diff
@@ -0,0 +1,34 @@
+name: busybox
+version: 1.0
+summary: busybox example to showcase the snapcraft kbuild plugin
+description:
+   this is a snapcraft example package that contains the output of the
+   snapcraft busybox example.
+
+apps:
+  ls:
+    command: ls
+    plugs: [old-security]
+  touch:
+    command: touch
+    plugs: [old-security]
```

Comment:
Just a nit, but can we name this something representative of what it is? Maybe `read-and-write-all` or something?